ante/evm/mono_decorator.go

```diff
@@ -126,6 +151,7 @@ func (md MonoDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bool, ne
 		// 5. signature verification
 		if err := SignatureVerification(
 			ethMsg,
+			ethTx,
```

Comment:
the reason for this, is `ethTx` caches the sender recovery result, so we avoid duplicate signature verification efforts.